Gruntfile.js

```diff
@@ -183,6 +183,14 @@ module.exports = function (grunt) {
         }
       }
     },
+    http: {
+      pattern_status: {
+        options: {
+           url: 'http://www.patternfly.org/patternfly-design/status/pattern-status.json'
```

Comment:
Would you mind changing this to https? If we leave it as http, secure pages will have complications.

Reply:
Can do!